src/App.js

```diff
@@ -11,6 +11,19 @@ import MessagesPage from "./MessagesPage";
 const App = () => {
   const [currentUserEmail, setCurrentUserEmail] = useState("");
 
+  const handleSignout = (event) => {
```

Comment:
what's the logic for moving this here?

Reply:
The signout handler gets passed as a prop to the Header component, and then changes the state of the App so that the page rerenders.

Reply:
That makes sense, thanks for explaining.